cmd/deploy_delete.go

```diff
@@ -45,6 +52,6 @@ func newDeleteDeploymentCmd(deployConfig *deployCommandConfig) *cobra.Command {
 		},
 	}
 
-	deleteDeploymentCmd.PersistentFlags().StringVarP(&deployConfigFile, "file", "f", "app-deploy.yaml", "The file name to use for the deployment configuration.")
+	deleteDeploymentCmd.PersistentFlags().StringVarP(&deployConfigFile, "file", "f", "app-deploy.yaml", "Filename of the deployment configuration you want to use.")
```

Comment:
We've used "deployment manifest" before, so should this be the same? We don't need "to use", because that's implied, but it might be worth explaining a bit about what it's used for, eg
"The name of the deployment manifest file for your application." or "Your application's deployment manifest." (if either of those make technical sense!)